README.md

```diff
@@ -43,11 +43,18 @@ It is a temporary live demo, all data will be deleted after 10 minutes. Sponsore
 docker run -d --restart=always -p 3001:3001 -v uptime-kuma:/app/data --name uptime-kuma louislam/uptime-kuma:1
 ```
 
-Uptime Kuma is now running on http://localhost:3001
+Uptime Kuma is now running on <http://0.0.0.0:3001>.
```

Comment:
I am still thinking, using `localhost` here could have more accessible for someone who would like to try this in their current PC. While `0.0.0.0` is not working for any cases.

Reply:
Not quite sure what case you are refering to.
I think you are referring to cases where the default route is not local or completely borked. Thing is, that I am not quite sure if localhost wou work in this case.

I assume that in somebody goes around and configures his*her default network route, they are knowlegable to know how to deal with this
=> likely does not matter

If you think this is better, I can revert this change ^^

https://superuser.com/questions/949428/whats-the-difference-between-127-0-0-1-and-0-0-0-0
does claim 

> **The all-zero value does have a special meaning.** So it is "valid", but has a meaning that may not be appropriate (and thus treated as not valid) for particular circumstances. **It is basically the "no particular address" placeholder.** For things like address binding of network connections, the result can be to assign an appropriate interface address to the connection. If you are using it to configure an interface, it can remove an address from the interface, instead. **It depends on the context of use to determine what "no particular address" really does.**
> 
> In the context of a route entry, it usually means the default route. That happens as a result more of the address mask, which selects the bits to compare. A mask of "0.0.0.0" selects no bits, so the compare will always succeed. So when such a route is configured, there is always somewhere for packets to go (if configured with a valid destination).

Reply:
Not quite sure what case you are refering to.
I think you are referring to cases where the default route is not local or completely borked. Thing is, that I am not quite sure if localhost wou work in this case.

I assume that in somebody goes around and configures his*her default network route, they are knowlegable to know how to deal with this
=> likely does not matter

If you think this is better, I can revert this change ^^

https://superuser.com/questions/949428/whats-the-difference-between-127-0-0-1-and-0-0-0-0
does claim 

> **The all-zero value does have a special meaning.** So it is "valid", but has a meaning that may not be appropriate (and thus treated as not valid) for particular circumstances. **It is basically the "no particular address" placeholder.** For things like address binding of network connections, the result can be to assign an appropriate interface address to the connection. If you are using it to configure an interface, it can remove an address from the interface, instead. **It depends on the context of use to determine what "no particular address" really does.**
> 
> In the context of a route entry, it usually means the default route. That happens as a result more of the address mask, which selects the bits to compare. A mask of "0.0.0.0" selects no bits, so the compare will always succeed. So when such a route is configured, there is always somewhere for packets to go (if configured with a valid destination).